php/formatting-functions.php

```diff
@@ -35,3 +35,30 @@ function wp_seo_no_formatting_tags_allowed( $string ) {
 
 	return $string;
 }
+
+/**
+ * Get the character count of a format string for display to users.
+ *
+ * @since 0.13.0
+ *
+ * @param  string $string String to count.
+ * @return string         The character count as a string, or a description of
+ *                        the count if it can't be calculated precicsely.
```

Comment:
The reason was partly so that the function would return only one type and partly to reinforce the idea that the function always returned descriptive text that sometimes happened to be a number. I tried to simplify the docs in 2af0307 and a837db1, but there might still be room for improvement from a fresh eyes. Let me know if I've misunderstood your request.